test/plugin/base_test.rb

```diff
@@ -1590,9 +1638,13 @@ def verify_default_log_entry_text(text, i, entry)
   end
 
   # The caller can optionally provide a block which is called for each entry.
-  def verify_json_log_entries(n, params, payload_type = 'textPayload')
+  def verify_log_entries(n, params, payload_type = 'textPayload')
+    jsonify_log_entries
     i = 0
     @logs_sent.each do |request|
```

Comment:
Might as well use `each_with_index` instead and avoid manually incrementing `i`.

Reply:
So it is. I was going to propose writing a wrapper to represent the nested loop, but it's probably not worth it at this point.

Reply:
Cool cool.